Library/Homebrew/cask/audit.rb

```diff
@@ -540,7 +541,33 @@ def audit_livecheck_version
       false
     end
 
-    def audit_livecheck_min_os
+    sig { void }
+    def audit_min_os
+      return unless online?
+
```

Comment:
```suggestion
      return unless online?
      return unless strict?

```

This doesn't look like a pattern the other cask audits use yet (probably because of the existing warning support we dropped) but: it feels like we should start doing it. Feels very wasteful to do all the work below but just ignore the results unless `strict?` later.